modules/stochastic_tools/test/tests/reporters/ActiveLearningGP/tests

```diff
@@ -15,22 +16,32 @@
       min_parallel = 2
       jsondiff = 'multiple_proc_multiple_row_ufunction.json'
       cli_args = "Samplers/mc/num_batch=2 Outputs/file_base='multiple_proc_multiple_row_ufunction'"
-      detail = 'multiple rows in the sampler using the U learning function;'
+      detail = 'multiple rows in the sampler using the U learning function and automatically increase the number of steps when re-training is needed;'
+    []
+    [SingleProc_SingleRow_Ufunction_retraining]
+      type = 'JSONDiff'
```

Comment:
You have increased the `learning_function_threshold` here to better demonstrate the step size increment. Is that right?

Reply:
It was to make sure re-training was needed